experiment_helpers/check_rmsThresh.m

```diff
@@ -55,9 +55,15 @@
             rmsValue = mean(data.rms(onset:offset, 1));
 
         % if no ost tracking, use RMS data to find onset/offset
-        elseif ~any(data.ost_stat >= 1) && any(data.rms(:, 1) > 0.03)
-            onset = find(data.rms > 0.01, 1, 'first') + 5;
-            offset = find(data.rms(:, 1)<0.03 & data.rms(:, 1)>0.02 & data.rms_slope<0, 1, 'first') - 5;
+        elseif any(data.rms(:, 1) > 0.025)
+            onset = find(data.rms > 0.025, 1, 'first') + 5;
+            offset = find(data.rms(:, 1)<0.03 & data.rms(:, 1)>0.015 & data.rms_slope<0, 1, 'first') - 5;
+            if isempty(offset)
+                offset = onset+10;
+            end
+            if offset > length(data.rms)
+                offset = length(data.rms);
+            end
 
             % use middle 80%
             onset = floor(onset + ((offset-onset)/10));
```

Comment:
FYI, consider replacing with midnperc(onset:offset,80)

Also, I can't comment on this because it's not part of the PR, but on line 65->71 below, are the parens correct? We want the first column of data.rms? (Or is the 1 meant to be an argument to "mean"?)

Reply:
Switched to using `midnperc` with commit c7c9caf.

Yes, we want to use the first column of data.rms. This is unchanged behavior from the original version of this function. Each column is a differently smoothed / filtered version of RMS coming out of Audapter.

Reply:
I took a second look and I realize our plotting should match the values we are using to calculate the rms. So we should either use your original code or otherwise redefine the variables `onset` and `offset` so the right interval gets plotted.

Can you remind me the justification for using the middle 80%? If we want to roughly matching the OST values for finding onset/offset, it's not clear to me why we should do this.